eng/pipelines/templates/stages/platform-matrix-conda-support.json

```diff
@@ -0,0 +1,10 @@
+{
+  "matrix": {
+    "Agent": {
+      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" },
```

Comment:
If you change these to something like `"ubuntu-18.04_conda"` then it will be easier to discern which jobs are which in the UI by the display name (which would then be `ubuntu_1804_conda_36`).